README.md

```diff
@@ -193,7 +193,7 @@ The package is written in Go and tested with Go 1.14, which can be downloaded fr
 Something like this should work for Debian / Ubuntu / Mint, but please check [gioui](https://gioui.org/) and general OpenGL docs to see requirements for your system:
 
 ```sh
-sudo apt-get install build-essential xorg-dev libgl1 libgl1-mesa-dev libgl1-mesa-glx redis-server
+sudo apt-get install build-essential xorg-dev libgl1 libgl1-mesa-dev libgl1-mesa-glx libgles2-mesa-dev redis-server
```

Comment:
While we are at it, let's either throw all prerequisites (i.e., `libxkbcommon-x11-0` and `libxkbcommon-x11-dev` too) in the same `install`.
Alternatively, do them separately on groups (e.g., `redis-server` by itself, all `org/mesa` together, and then `libxkbcommon`) to better show what are the dependencies.

What do you think?

Reply:
Sure, let's do it

Reply:
Nice, merge away!